src/style.css

```diff
@@ -0,0 +1,110 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+}
+
+body {
+  background-color: rgba(214, 214, 214, 0.858);
+  font-family: 'Lato', sans-serif;
+}
+
+main {
+  width: 90%;
+  margin: 0 auto;
+  background-color: rgba(214, 214, 214, 0.858);
+}
+
+.todo-section {
+  margin-block: 50px;
+  background-color: whitesmoke;
+  max-width: calc(100vh - 50px);
+  box-shadow: 0 0 10px 4px rgba(85, 85, 85, 0.2);
+}
+
+.todo-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2.5%;
+  border-bottom: 0.3px solid rgba(119, 119, 119, 0.3);
+}
+
+form {
+  display: flex;
+  justify-content: space-between;
+  padding: 2.5%;
+  border-bottom: 0.3px solid rgba(119, 119, 119, 0.3);
+}
+
+form input {
+  width: 99%;
+  border: none;
+  outline: none;
+  background-color: transparent;
+  font-size: 1.3rem;
+}
+
+form input::placeholder {
+  font-style: italic;
+  font-weight: 100;
+  font-size: 1.2rem;
+}
+
+form button {
+  border: none;
+  background-color: transparent;
+  outline: none;
+  cursor: pointer;
+}
+
+.todo {
+  display: flex;
+  gap: 10px;
+  padding: 2.5%;
+  align-items: center;
+  border-bottom: 0.3px solid rgba(119, 119, 119, 0.3);
+}
+
+.todo.completed {
+  background-color: antiquewhite;
+}
+
+.checkbox {
+  width: 20px;
+  height: 20px;
+  cursor: pointer;
+}
+
+.todo-label {
+  width: 93%;
+  font-size: 1.2rem;
+}
+
+.todo-label.completed {
+  text-decoration: line-through;
+  color: rgb(124, 124, 124);
+}
+
+.fa-trash-alt {
+  display: none;
+}
+
+.clear-complete-container {
+  padding: 5% 2.5%;
+  text-align: center;
+  font-size: 1.2rem;
+  background-color: rgb(223, 223, 223);
+}
+
+.clear-completed:hover {
+  text-decoration: underline;
+}
```

Comment:
- [ OPTIONAL]  I recommend you to style your `completed text` with a little bit of gray color not black, as it is styled from the provided design from the videos.  You should style it black only if you are hovering over the text, as it is showing from the reference [vidoes ](https://www.youtube.com/watch?v=AcUd-_Yjjqg)
![image](https://user-images.githubusercontent.com/43877689/184193870-51122656-43bd-45b3-9414-4e6a13d75b55.png)
This will make your project look similar to the reference design provided from the vidoes